### PR TITLE
Added the process graph working with opendatacube backend

### PR DESCRIPTION
**Pull request by clausmichele**
I'm not using merge_cubes with overlap resolver, instead I rename the bands before merging and use reduce_dimension with array_element after merge_cubes. I created this with VITO web editor, but unfortunately if I copy and paste it into a new window with a new instance it doesn't display it.
This graph still misses the last part with the MODIS data, which needs resample spatial and temporal.

### CODE REVIEW

File: eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json [hunk 1]
```diff
@@ -0,0 +1,746 @@
...
+    "1": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "31"
+        },
+        "cube1": {
+          "from_node": "10"
+        },
+        "overlap_resolver": {
```
Comment (m-mohr) [inline eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json:12]: The empty overlap resolver is why the web editor complains, not sure why that got generated multiple times.

File: eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+    "4": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube2": {
+          "from_node": "25"
+        },
+        "cube1": {
+          "from_node": "35"
+        },
+        "overlap_resolver": {
```
Comment (m-mohr) [inline eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json:45]: Here's still a multiply overlap_resolver. Is that intentional?

File: eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json (hunk 1 above)
Comment (m-mohr) [reply eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json]: see #115

File: eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json (hunk 2 above)
Comment (clausmichele) [reply eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json:45]: No I just forgot it, let me remove it.

**Comment by m-mohr** [comment]
I'm not sure why and what the two commits do, but it still shows the empty parameters. Seems they have been added again. Could you somehow fix that again?

File: eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+              "process_id": "min",
+              "arguments": {
+                "data": {
+                  "from_parameter": "data"
+                }
+              },
+              "result": true
+            }
+          },
+          "parameters": [
```
Comment (m-mohr) [inline eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json:111]: All these "empty" parameters should be removed.

Comment (m-mohr) [reply eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json:111]: see also #10

**Comment by clausmichele** [comment]
I'm trying to use GitHub Desktop app, but it's so confusing, I'll fix it tomorrow.

File: eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json (hunk 3 above)
Comment (clausmichele) [reply eurac_wetsnow_usecase/eurac_wetsnow_processgraph_v1.0_opendatacube.json]: I fixed it, now should be fine.

**Pull request merged**